server/tools/templates/cytometry.html

```diff
@@ -33,7 +33,7 @@
     <label for="id_study_0" style="float:left;">
       <span>Select Study</span>
       <input class="autocomp autocomp_study" id="id_study_0" name="study_0" type="text"
-          eddautocompletetype="StudyWritable" />
+          edd-autocompletetype="StudyWritable" />
```

Comment:
```suggestion
          data-eddautocompletetype="StudyWritable" />
```

This looks like a simple mechanical mistake, after all the others got the `data-` prefix.